examples/nerdctl-ipfs-registry-kubernetes/ipfs-cluster/bootstrap.yaml.sh

```diff
@@ -28,7 +28,7 @@ done
 
 TMPIDFILE=$(mktemp)
 BOOTSTRAP_KEY=$(ipfs-key 2>"${TMPIDFILE}" | base64 -w 0)
-ID=$(cat "${TMPIDFILE}" | grep "ID " | sed -E 's/[^:]*: (.*)/\1/')
+ID=$(< "${TMPIDFILE}" grep "ID " | sed -E 's/[^:]*: (.*)/\1/')
```

Comment:
`grep "ID " "$TMPIDFILE"` might be more readable / easier to understand for most folks

Reply:
Well, I agree. :D